File: src/main/java/com/ultreon/mods/screenshotmanager/text/CommonTexts.java
package com.ultreon.mods.screenshotmanager.text;

import net.minecraft.network.chat.Component;
import net.minecraft.network.chat.TranslatableComponent;

public class CommonTexts {
    public static final TranslatableComponent PREV = new TranslatableComponent("misc.screenshotmgr.prev");
    public static final TranslatableComponent NEXT = new TranslatableComponent("misc.screenshotmgr.next");
    public static final TranslatableComponent NO_SCREENSHOTS = new TranslatableComponent("screen.screenshotmgr.no_screenshots");
    public static final TranslatableComponent ERROR_OCCURRED = new TranslatableComponent("screen.screenshotmgr.error_occurred");
    public static final TranslatableComponent INVALID_SCREENSHOT = new TranslatableComponent("screen.screenshotmgr.invalid_screenshot");

    public static TranslatableComponent loadedScreenshots(int loaded, int total) {
        return new TranslatableComponent("screen.screenshotmgr.loaded_screenshots", loaded, total);
    }
}

File: src/main/java/com/ultreon/mods/screenshotmanager/text/CommonTexts.kt
package com.ultreon.mods.screenshotmanager.text

import dev.ultreon.quantum.text.MutableText
import dev.ultreon.quantum.text.TextObject

object CommonTexts {
    val loading: MutableText = TextObject.translation("screen.screenshotmgr.loading")
    val prev: MutableText = TextObject.translation("misc.screenshotmgr.prev")
    val next: MutableText = TextObject.translation("misc.screenshotmgr.next")
    val noScreenshots: MutableText = TextObject.translation("screen.screenshotmgr.no_screenshots")
    val errorOccurred: MutableText = TextObject.translation("screen.screenshotmgr.error_occurred")
    val invalidScreenshot: MutableText = TextObject.translation("screen.screenshotmgr.invalid_screenshot")
}